testprojects/src/python/vcs_test/changed_integration_test.py

```diff
@@ -326,143 +252,3 @@ def test_lines_one_line_on_the_edge_deleted(repo: str) -> None:
         changed_since="HEAD~1",
         extra_args=["--enable-target-origin-sources-blocks"],
     )
-
-
-def test_unowned_file(repo: str) -> None:
-    """Unowned files should still work with target-less goals like `count-loc`.
-
-    If a file was removed, the target-less goals should simply ignore it.
-    """
-    create_file("dir/some_file.sh", "# blah")
-    assert_count_loc(repo, expected_num_files=1)
-    assert_list_stdout(repo, [])
-
-    delete_file("dir/some_file.sh")
-    assert_count_loc(repo, expected_num_files=0)
-
-
-def test_delete_generated_target(repo: str) -> None:
-    """If a generated target is deleted, we claim the target generator was modified.
-
-    It's unlikely these are actually the semantics we want...See:
-    * https://github.com/pantsbuild/pants/issues/13232
-    * https://github.com/pantsbuild/pants/issues/14975
-    """
-    delete_file("transitive.sh")
-    for dependents in DependentsOption:
-        assert_list_stdout(repo, ["//:lib"], dependents)
-        assert_count_loc(repo, dependents, expected_num_files=2)
-
-    # Make sure that our fix for https://github.com/pantsbuild/pants/issues/15544 does not break
-    # this test when using `--tag`.
-    for dependents in (DependentsOption.NONE, DependentsOption.TRANSITIVE):
-        assert_list_stdout(repo, ["//:lib"], dependents, extra_args=["--tag=a"])
-        assert_count_loc(repo, dependents, expected_num_files=1, extra_args=["--tag=a"])
-
-    # If we also edit a sibling generated target, we should still (for now at least) include the
-    # target generator.
-    append_to_file("app.sh", "# foo")
-    for dependents in DependentsOption:
-        assert_list_stdout(repo, ["//:lib", "//app.sh:lib"], dependents)
-        assert_count_loc(repo, dependents, expected_num_files=2)
-
-
-def test_delete_atom_target(repo: str) -> None:
-    delete_file("standalone.sh")
-    assert_list_stdout(repo, ["//:standalone"])
-
-    # The target-less goal code path will trigger checking that `sources` are valid.
-    result = _run_pants_goal(repo, "count-loc")
-    result.assert_failure()
-    assert "must have 1 file, but it had 0 files." in result.stderr
-
-
-def test_change_build_file(repo: str) -> None:
-    """Every target in a BUILD file is changed when it's edited.
-
-    This is because we don't know what the target was like before-hand, so we're overly
-    conservative.
-    """
-    append_to_file("BUILD", "# foo")
-    # Note that the target generator `//:lib` does not show up.
-    assert_list_stdout(
-        repo,
-        [
-            "//:standalone",
-            "//:tables#companies",
-            "//:tables#employees",
-            "//:tables#persons",
```

Comment:
@kaos I believe this test does actually check the BUILD file and it is unique

Reply:
did I remove too much.. 👀 ?

Reply:
actually, I don't feel we need to keep this in the test, as it doesn't relate to the change we're introducing here.

Reply:
My personal opinion that it's important, because it checks that generated targets are included in the dependencies list, but the generator target `//:tables` is not included. If it were included that would mean that the granularity feature doesn't really do anything, because all the dependents get invalidated when any `table` changes

Reply:
I looked closely and I think we can leave it out, other tests indirectly check the same thing